sdk/tests/conformance2/context/methods-2.html

```diff
@@ -247,11 +249,7 @@
 "isSampler",
 "bindSampler",
 "samplerParameteri",
-"samplerParameteriv",
-"samplerParameteriv",
 "samplerParameterf",
-"samplerParameterfv",
-"samplerParameterfv",
```

Comment:
I think we should leave these entry points in place. Even though they're not necessary right now, if for example a vendor exposes a texture border color extension, we'd have to add these entry points back in that extension. What do you think?